pages/glossary/visible-inner-text.md

```diff
@@ -0,0 +1,38 @@
+# Visible inner text
+
+(The "visible inner text" defined here is similar to, but not the same as, [visible text content][].)
```

Comment:
This could also refer to the inner text algorithm from HTML, since it is also quite similar but a bit different.

Reply:
I pushed a commit for this a minute ago.